src/pages/mentorship-3.js

```diff
@@ -0,0 +1,294 @@
+import { useState } from 'react';
+import Head from 'next/head';
+
+import { mentorsData } from '../data/mentorship-2-mentors';
+
+import Accordion from '../components/Accordion';
+import Timeline from '../components/Timeline';
+import TimelineItem from '../components/TimelineItem';
+import MentorsCard from '../components/MentorsCard'
+import Gallery from '../components/Gallery';
+import TestimonialCard from '../components/TestimonialCard';
+
+import mentorshipTimeline from '../data/mentorship-timeline-3';
+import programsFAQ from '../data/mentorship-faq.js';
+import mentorshipTestimonials from '../data/mentorship-testimonials.js';
+
+import Modal from '../components/Modal';
+
+export default function Home () {
+  const [showMentorsModal, setShowMentorsModal] = useState(false)
+  const [selectedTopic, setSelectedTopic] = useState(null)
+
+  const handleViewMentors = (e) => {
+    setSelectedTopic(e.target.dataset.message)
+    setShowMentorsModal(true)
+  }
+
+  return (
+    <div>
+      <Head>
+        <title>Mentorship #3 | TechLadies</title>
+      </Head>
+      <main>
+        <div className="container py-6">
+          <div className="row gy-6">
+            <div className="col-md-6">
+              <h1 class="large-line-height">TechLadies Mentorship</h1>
+              <p className="mb-4 text-body large-line-height">
+                TechLadies Mentorship is a 12-week program that provides women with personal career guidance
+                from experienced industry practitioners as they grow their
+                career in Software Engineering, UI/UX Design, Data
+                Science/Engineering/Analysis, Product Management.
+                <br />
+                <br />
+                Registration for the TechLadies Mentorship Program 2021 has just opened. Sign up for this year’s Mentorship Program now!
+              </p>
+              <div className="row">
+                <div className="col-md-6">
+                  <a className="btn btn-primary" target="_blank" href="https://forms.gle/2a1Ns12WSV5tJJ3b7">
+                      Sign up as Mentor!
+                  </a>
+                </div>
+               <div className="col-md-6">
+                  <a className="btn btn-primary" target="_blank" href="https://forms.gle/5xoBrmBPyNNuqpgo9">
+                      Sign up as Mentee!
+                  </a>
+               </div>
+              </div>
+              
+            </div>
+            <div className="col-md-5  offset-md-1">
+              <img
+                alt="TODO: REPLACE ME"
```

Comment:
do you want to replace this? :D

Reply:
yeah im not sure what to replace with?

Reply:
It's just alternative text in case the image itself fails to load - usually a short description of the image
https://www.w3schools.com/tags/att_img_alt.asp

Maybe for this one, just "mentorship illustration" would be ok :)